requirements/requirements.txt

```diff
@@ -47,7 +47,6 @@ scipy==1.7.3;python_version=='3.7'
 zipfile-deflate64==0.2.0
 
 # docs
-ipywidgets==7.7.0
```

Comment:
I added this once upon a time because I ran into issues locally, but the doc tests seem to pass without it and it isn't a direct dep, so I removed it